Grid/serialisation/Hdf5IO.h

```diff
@@ -204,9 +268,9 @@ namespace Grid
     }
     
     // read the flat vector
-    std::vector<Element> buf(size);
-
-    if (size > dataSetThres_)
+    buf.resize(size);
+    
+    if (size * sizeof(Element) > dataSetThres_)
```

Comment:
Same than previous comment.

Reply:
My bad! This is now back the way it was.